.travis.yml

```diff
@@ -45,6 +45,14 @@ jobs:
         - tox -- tests/ --ignore tests/test_pythonpackage.py
         # (we ignore test_pythonpackage.py since these run way too long!!
         #  test_pythonpackage_basic.py will still be run.)
+      name: "Tox Pep8"
+      env: TOXENV=pep8
```

Comment:
Here we use a tox feature, explained [here](https://tox.readthedocs.io/en/latest/example/general.html#selecting-one-or-more-environments-to-run-tests-against)